synapse/cortex.py

```diff
@@ -1649,6 +1649,9 @@ async def getDeprLocks(self):
         return retn
 
     async def _warnDeprLocks(self):
+        if __debug__ and not s_common.envbool('SYN_CORTEX_WARN_DEPRLOCKS'):
```

Comment:
lets use `SYNDEV_WARN_DEPRLOCKS` since its really a dev/test knob and doesn't get confused with config options.